public/js/app.js

```diff
@@ -0,0 +1 @@
+window.bootstrap = require('bootstrap/dist/js/bootstrap.bundle.js');
```

Comment:
Why are you assigning this to the `window`?

Reply:
I had really issues setting this up, no `import 'bootstrap'` or similar works. So if followed this recommendation: https://stackoverflow.com/a/65260998/1058552